plugins/main/bridge/bridge.go

```diff
@@ -791,28 +791,35 @@ func cmdDel(args *skel.CmdArgs) error {
 		return err
 	})
 	if err != nil {
-		//  if NetNs is passed down by the Cloud Orchestration Engine, or if it called multiple times
+		// if NetNs is passed down by the Cloud Orchestration Engine, or if it called multiple times
 		// so don't return an error if the device is already removed.
 		// https://github.com/kubernetes/kubernetes/issues/43014#issuecomment-287164444
 		_, ok := err.(ns.NSPathNotExistErr)
-		if ok {
-			return ipamDel()
+		if !ok {
+			return err
 		}
-		return err
 	}
 
-	// call ipam.ExecDel after clean up device in netns
-	if err := ipamDel(); err != nil {
-		return err
-	}
+	if len(ipnets) == 0 {
+		// could not get ip address within netns, so try to get it from prevResult
```

Comment:
nit: this comment is a bit misleading; we don't get the IP from the interface in the netns. We just use PrevResult

Reply:
nevermind, I'm wrong :-)